app/forms/scholars_archive/default_work_form_behavior.rb

```diff
@@ -14,19 +14,19 @@ module DefaultWorkFormBehavior
+      self.terms += [:nested_related_items, :nested_ordered_creator, :nested_ordered_title, :nested_ordered_abstract, :nested_ordered_additional_information, :nested_ordered_contributor, :date_uploaded, :date_modified, :doi, :other_affiliation, :academic_affiliation, :alt_title, :license, :resource_type, :date_available, :date_copyright, :date_issued, :date_collected, :date_valid, :date_reviewed, :date_accepted, :degree_level, :degree_name, :degree_field, :replaces, :nested_geo, :hydrologic_unit_code, :funding_body, :funding_statement, :in_series, :tableofcontents, :bibliographic_citation, :peerreviewed, :digitization_spec, :file_extent, :file_format, :dspace_community, :dspace_collection, :isbn, :issn, :embargo_reason, :conference_name, :conference_section, :conference_location]
 
       self.terms -= [:creator, :title]
       self.required_fields += [:resource_type, :nested_ordered_creator, :nested_ordered_title]
-      self.required_fields -= [:keyword, :creator, :title]
+      self.required_fields -= [:keyword, :creator, :title, :contributor]
 
       def primary_terms
-        [:nested_ordered_title, :alt_title, :nested_ordered_creator, :contributor, :abstract, :license, :resource_type, :doi, :dates_section, :degree_level, :degree_name, :degree_field, :bibliographic_citation, :academic_affiliation, :other_affiliation, :in_series, :subject, :tableofcontents, :rights_statement] | super
+        [:nested_ordered_title, :alt_title, :nested_ordered_creator, :nested_ordered_contributor, :nested_ordered_abstract, :license, :resource_type, :doi, :dates_section, :degree_level, :degree_name, :degree_field, :bibliographic_citation, :academic_affiliation, :other_affiliation, :in_series, :subject, :tableofcontents, :rights_statement] | super
       end
 
       def secondary_terms
-        t = [:nested_related_items, :hydrologic_unit_code, :geo_section, :funding_statement, :publisher, :peerreviewed, :conference_name, :conference_section, :conference_location, :language, :file_format, :file_extent, :digitization_spec, :replaces, :additional_information, :isbn, :issn]
-        t << [:keyword, :source, :funding_body, :dspace_community, :dspace_collection, :description, :identifier] if current_ability.current_user.admin?
+        t = [:nested_related_items, :hydrologic_unit_code, :geo_section, :funding_statement, :publisher, :peerreviewed, :conference_name, :conference_section, :conference_location, :language, :file_format, :file_extent, :digitization_spec, :replaces, :nested_ordered_additional_information, :isbn, :issn]
+        t << [:keyword, :source, :funding_body, :dspace_community, :dspace_collection, :nested_ordered_additional_information, :identifier] if current_ability.current_user.admin?
```

Comment:
It looks like this line removes `:description` from the form. I remember @simholt mentioned that we should still keep it in the form for admin users only. I think we can leave it as:
```
t << [:keyword, :source, :funding_body, :dspace_community, :dspace_collection, :description, :identifier] if current_ability.current_user.admin?
```